src/types/ellipse.js

```diff
@@ -77,6 +77,11 @@ function pointInEllipse(p, ellipse) {
   if (xRadius <= 0 || yRadius <= 0) {
     return false;
   }
-
-  return (Math.pow(p.x - center.x, 2) / Math.pow(xRadius, 2)) + (Math.pow(p.y - center.y, 2) / Math.pow(yRadius, 2)) <= 1.0;
+  // https://stackoverflow.com/questions/7946187/point-and-ellipse-rotated-position-test-algorithm
+  const angle = toRadians(rotation || 0);
+  const cosAngle = Math.cos(angle);
+  const sinAngle = Math.sin(angle);
+  const a = Math.pow(cosAngle * (p.x - center.x) + sinAngle * (p.y - center.y), 2);
+  const b = Math.pow(sinAngle * (p.x - center.x) - cosAngle * (p.y - center.y), 2);
+  return (a / Math.pow(xRadius, 2)) + (b / Math.pow(yRadius, 2)) <= 1;
```

Comment:
Do the width and height need to be adjusted?

Reply:
No, they don't. At least I don't think so. Why?

Reply:
I was thinking it should be the rotated xRadius/yRadius that we need to compare against, but is it?

I think this one would deserve some dedicated tests.

Reply:
in my opinion, width and height should be affected by a rotation, because they are remaining the same. But I could be wrong.
In fact, in the stackoverflow, where I took the solution, the width and height are not touched by rotation.

Reply:
> I think this one would deserve some dedicated tests.

added specific test case on rotated ellipse